moped-toolbox/populate_geometry_transform/transpose_geometry.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/python3
+
```

Comment:
i believe we'll want to merge #792/main into this branch, there are some shared files (like this one) which are more current on main. this is not currently reflected in this diff...

Reply:
I agree John. I've just now merged `main` into this branch, which went in without conflict. ~470 commits from main made it into here.

Thanks for thinking of this!